wolfssl/wolfcrypt/settings.h

```diff
@@ -469,6 +472,23 @@
     #include <nx_api.h>
 #endif
 
+
+#ifdef NDS
+    #include <stddef.h>
+    #define SIZEOF_LONG_LONG 8
+    #define socklen_t int
+    #define IPPROTO_UDP 17
+    #define IPPROTO_TCP 6
+
+    /* Libnds doesn't include sys/uio.h.  */
+    /* Structure for scatter/gather I/O.  */
+    struct iovec
```

Comment:
Note: This can also be resolved by `#define NO_WRITEV`.

Reply:
Thanks, I changed it.